src/wrapped/wrappedgtk3_private.h

```diff
@@ -2828,7 +2828,7 @@ GO(gtk_style_context_get_scale, iFp)
 GO(gtk_style_context_get_screen, pFp)
 GO(gtk_style_context_get_section, pFpp)
 GO(gtk_style_context_get_state, uFp)
-//GO2(gtk_style_context_get_style, vFpV, gtk_style_context_get_style_valist)
+GO2(gtk_style_context_get_style, vFpV, gtk_style_context_get_style_valist)
```

Comment:
No, that doesn't work like that on x86_64, only on x86 will this trick work (that's why it was commented)